install_cloudnine.sh

```diff
@@ -0,0 +1,23 @@
+git checkout -b main
+git pull -f origin main
+rvm install ruby-2.4.4
+source $(rvm 2.4.4 do rvm env --path)
+rvm use ruby-2.4.4
+gem install rails -v 3.2.0
+sudo apt-get update
+sudo apt-get -y install bundler libmysqlclient-dev imagemagick ruby-rmagick libfreeimage3 libfreeimage-dev ruby-dev libmagickcore-dev libmagickwand-dev
+# exporter-only:
+#sudo apt-get install gdal-bin python-gdal curl libcurl4-openssl-dev libssl-dev zip
+gem install bundler
+bundle _1.16.2_ install
+cp db/schema.rb.example db/schema.rb
+cp config/database.yml.cloud9.example config/database.yml
```

Comment:
Strangely i didn't see that this completed successfully, and `database.yml` was empty! 😦

Reply:
I think the repo just doesn't have a `config/database.yml.cloud9.example`

Reply:
Told you @jywarren :point_right:  https://github.com/publiclab/mapknitter/pull/420#issuecomment-475982087

Reply:
Actually, @sashadev-sky it's created by @jywarren here- https://github.com/publiclab/mapknitter/pull/420/commits/db4e69bc40013ba0e6dff9b502e0efce548641c3 :smile:

Reply:
@gauravano oops sorry! Also didn't see that you already commented that 5 days ago 🙈